python/ray/serve/controller.py

```diff
@@ -766,6 +766,10 @@ def list_deployments(self, include_deleted: Optional[bool] = False) -> bytes:
             )
         return deployment_route_list.SerializeToString()
 
+    def list_deployment_names(self) -> List[str]:
+        """Gets the current list of all deployments names."""
```

Comment:
```suggestion
        """Gets the current list of all deployments' names."""
```

Could you also document what format these names are (i.e. `[app_name]_[deployment_name]` or just `[deployment_name]`)?